Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -380,9 +387,44 @@ def checkFileLength (lines : Array String) (existingLimit : Option ℕ) : Option
 
 end
 
+
+section unicodeLinter
+
+/- Hashable instance for use with the unicodeLinter -/
```

Comment:
Does this fix it? A /- Comment -/ is just a multi-line comment in Lean, a /-- Comment -/ is a documentation comment.
```suggestion
/- Hashable instance for use with the unicodeLinter -/